cypress/support/commands.js

```diff
@@ -1,14 +1,20 @@
 import '@testing-library/cypress/add-commands';
 import { getIfExists } from '../helpers';
+
+const HOSTNAME = Cypress.env('BACKEND_HOST') || 'localhost';
+const GUILLOTINA_API_URL = `http://${HOSTNAME}:8081/db/web`;
+const PLONE_SITE_ID = Cypress.env('SITE_ID') || 'plone';
+const PLONE_API_URL = `http://${HOSTNAME}:55001/${PLONE_SITE_ID}`;
```

Comment:
Can we introduce another fallback using another env var? like
```suggestion
const PLONE_API_URL = Cypress.env('API_PATH') || `http://${HOSTNAME}:55001/${PLONE_SITE_ID}`;
```

Since matches the others env vars naming and I introduced it in https://github.com/plone/volto/pull/3242/files#diff-70c7e133120d2e4b9afe435052f4988b27e00f9423e2b22e5b4d09f6d58e04d3R2

Thanks!